packages/wrangler/src/publish/publish.ts

```diff
@@ -486,6 +486,7 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 					: []
 			),
 			vars: { ...config.vars, ...props.vars },
+			script_metadata_binding: config.script_metadata_binding,
```

Comment:
It wont let me link directly to the line so here is the addition in the file you mentioned above @danielrs

Reply:
Oh I see what I missed

Reply:
Nice catch 😄